libs/utils/src/Card/memory_card.ts

```diff
@@ -1,160 +0,0 @@
-import { Result, ok } from '@votingworks/basics';
-import {
-  Card,
-  CardSummary,
-  Optional,
-  safeParseJson,
-  ShortAndLongValues,
-} from '@votingworks/types';
-import { z } from 'zod';
-
-/* eslint-disable @typescript-eslint/require-await */
-
-/**
- * Implements the `Card` API with an in-memory implementation.
- */
-export class MemoryCard implements Card {
```

Comment:
I got rid of this `MemoryCard` implementation of the old card API because:

1. We no longer use it anywhere (it was previously used in tests, but application-level tests now mock auth).
2. Its name can be quite confusing. It's not referring to an actual memory card. It's referring to a mock of a card that uses computer memory.